examples/plugins/cmd-runner/README.md

```diff
@@ -0,0 +1,27 @@
+# Command Runner
+
+This example creates an Sonobuoy plugin which runs each of its arguments (via /bin/sh) and saves the output in a results file.
+
+Kubectl is added to the image so that it can grab information from/about the cluster.
+
+All of the results files are added into a tar file and then the 'done' file is written, telling the Sonobuoy worker where to find the output.
```

Comment:
We should document this process a bit, then link to it. I think the story still isn't told very well in our docs.

Reply:
Linking to https://github.com/heptio/sonobuoy/blob/master/docs/plugins.md

Was tempted to link directly to https://github.com/heptio/sonobuoy/blob/master/docs/plugins.md#contract but as we edit that file I dont want to break links just because we changed a header.